common/src/main/resources/assets/wynntils/lang/zh_tw.json

```diff
@@ -1,982 +1,1001 @@
-  "feature.wynntils.lootrun.cycleDistance.name": "彩色路徑循環長度",
-  "feature.wynntils.lootrun.name": "寶藏路徑",
-  "feature.wynntils.lootrun.pathType.description": "路徑要怎樣渲染？",
-  "feature.wynntils.lootrun.pathType.name": "路徑渲染樣",
-  "feature.wynntils.lootrun.rainbowLootRun.description": "路徑的顏色要是否像彩虹嗎？",
-  "feature.wynntils.lootrun.rainbowLootRun.name": "渲染路徑像彩虹",
-  "feature.wynntils.lootrun.recordingPathColor.description": "記錄著的路徑要渲染什麼顏色？",
-  "feature.wynntils.lootrun.recordingPathColor.name": "記錄的路徑色",
-  "feature.wynntils.lootrun.showNotes.description": "要是否顯示活性寶藏路徑的記錄？",
-  "feature.wynntils.lootrun.showNotes.name": "寶藏路徑記錄",
-  "feature.wynntils.lootrunUtils.addedNote": "加了記錄在 %s，有內容文本: ",
-  "feature.wynntils.lootrunUtils.chestAdded": "加了箱子，地點在 %s。",
-  "feature.wynntils.lootrunUtils.chestAlreadyAdded": "在 %s 的箱子已經被加了。",
-  "feature.wynntils.lootrunUtils.chestDoesNotExist": "在 %s 的箱子不再寶藏路徑裡。",
-  "feature.wynntils.lootrunUtils.chestRemoved": "減了箱子，地點在 %s。",
-  "feature.wynntils.lootrunUtils.clearSuccessful": "清除了現在活性/紀錄的路徑",
-  "feature.wynntils.lootrunUtils.errorSavingLootrun": "有故障在保存著路徑時。",
-  "feature.wynntils.lootrunUtils.errorSavingLootrunAlreadyExists": "有這名的寶藏路徑已經存在。",
-  "feature.wynntils.lootrunUtils.listClickToDelete": "點擊這裡刪除這位紀錄。",
-  "feature.wynntils.lootrunUtils.listNoteHeader": "寶藏路徑的紀錄: ",
-  "feature.wynntils.lootrunUtils.listNoteNoNote": "現在的路徑沒有紀錄。",
-  "feature.wynntils.lootrunUtils.lootrunCouldNotBeDeleted": "路徑 \"%s\" 刪不了。",
-  "feature.wynntils.lootrunUtils.lootrunCouldNotBeLoaded": "路徑 \"%s\" 加載不了。",
-  "feature.wynntils.lootrunUtils.lootrunCouldNotBeRenamed": "路徑 \"%s\" 改名到 \"%s\" 不了。",
-  "feature.wynntils.lootrunUtils.lootrunDeleted": "刪了爆讚路徑 \"%s\"",
-  "feature.wynntils.lootrunUtils.lootrunDoesntExist": "爆讚路徑 \"%s\" 不存在。",
-  "feature.wynntils.lootrunUtils.lootrunRenamed": "改名路徑 \"%s\" 到 \"%s\"。",
-  "feature.wynntils.lootrunUtils.lootrunStart": "爆讚路徑在 %s %s %s 的座標。",
-  "feature.wynntils.lootrunUtils.noActiveLootrun": "沒有活性或紀錄的路徑",
-  "feature.wynntils.lootrunUtils.notRecording": "您現在不在紀錄一位路徑。",
-  "feature.wynntils.lootrunUtils.noteRemovedSuccessfully": "刪了在 %s 有內容文本 \"%s\" 的紀錄。",
-  "feature.wynntils.lootrunUtils.noteUnableToFind": "找不見在 %s 的紀錄。",
-  "feature.wynntils.lootrunUtils.recordStart": "開始紀錄一位寶藏路徑。執行 %s 如要停紀錄。",
-  "feature.wynntils.lootrunUtils.recordStop1": "停了紀錄一位寶藏路徑。執行 %s 如要刪這位。",
-  "feature.wynntils.lootrunUtils.recordStop2": "執行 %s 如要保存這位。",
-  "feature.wynntils.lootrunUtils.savedLootrun": "保存了寶藏路徑。",
-  "feature.wynntils.lootrunUtils.undoNotFarEnough": "您沒有動足夠，無法遠撤銷路徑的一部。",
-  "feature.wynntils.lootrunUtils.undoStandNear": "您對路徑太遠，無法撤銷路徑的一部。",
-  "feature.wynntils.lootrunUtils.undoSuccessful": "撤銷了路徑的一部。",
-  "feature.wynntils.lowHealthVignette.animationSpeed.description": "如果在用脈衝動畫，脈衝的速度是多少？",
+  "feature.wynntils.itemTextOverlay.amplifierTierEnabled.description": "是否應該繪製一個顯示 Corkian Amplifier 品階的物品覆疊層?",
+  "feature.wynntils.itemTextOverlay.amplifierTierEnabled.name": "科金放大器品階格式",
```

Comment:
Translating ”Corkian" seems a bit odd

Reply:
I was thinking about should I translate this or not. If I don't translate, this item will be half is English, half is Chinese, might look wierd. So I decided to translate the full item name,and left the `xxx.description` being English.